packages/skpm/src/commands/publish.js

```diff
@@ -124,16 +124,23 @@ export default asyncCommand({
 
     let tag = skpmConfig.version
 
-    const spinner = ora({
-      text: `Checking if \`${repo}\` is accessible`,
-      color: 'magenta',
-    }).start()
+    let spinner = new ora()
 
     function print(text) {
       if (process.env.CI) {
         console.log(text)
       } else {
-        spinner.text = text
+        if (spinner) {
```

Comment:
Refactor `print()` by adding functionality to start spinner again, creating a new instance if necessary, or simply setting `text` if spinner is spinning.